book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Please add Getters and Setters(attr_accessor) for @rentals instead Getters only. So the add rental method below can create a new rental.